addon/components/cfb-form-editor/question.js

```diff
@@ -135,6 +164,10 @@ export default Component.extend(ComponentQueryManager, {
     return model;
   }),
 
+  namespace: computed("options.namespace", function() {
+    return slugify(this.options.get("namespace") || "");
```

Comment:
You could also do something like 
```suggestion
    return slugify(this.options.get("namespace")) || null;
```
Or is `slugify` throwing if undefined is passed? If yes, then you'd have to keep `|| ""` but could still add the `|| null`.

Reply:
Yeah. Slugify needs a string. Otherwise, it throws at `.toLowerCase()`.
https://github.com/projectcaluma/ember-caluma/blob/master/addon/utils/slugify.js#L4

Reply:
Oh wow, I'm quite surprised that we implimented this ourselves in the first place. Anyway, in that case it would be 
```suggestion
    return slugify(this.options.get("namespace") || "") || null;
```

Reply:
Or you just patch slugify to return `undefined` if no argument is given :wink:

Reply:
Why `null`?